packages/react-composites/src/composites/CallComposite/pages/LobbyPage.tsx

```diff
@@ -76,3 +109,17 @@ const overlayPropsWaitingToBeAdmitted = (strings: CallCompositeStrings): LobbyOv
   moreDetails: strings.lobbyScreenWaitingToBeAdmittedMoreDetails,
   overlayIcon: <CallCompositeIcon iconName="LobbyScreenWaitingToBeAdmitted" />
 });
+
+/* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
```

Comment:
Remove this conditional compilation directive. This function will be used unconditionally if you accept the suggestion above.